app/controllers/spaces_controller.rb

```diff
@@ -1,88 +1,64 @@
 # frozen_string_literal: true
 
 class SpacesController < ApplicationController
-  before_action :set_space, only: %i[show edit update destroy]
+  before_action :set_space, only: %i[ show edit update destroy ]
   before_action :check_multi_tenant_mode, only: %i[new index]
-  before_action :set_space_and_user_roles, only: %i[create]
 
-  # GET /spaces or /spaces.json
   def index
     @spaces = current_user.spaces.page params[:page]
   end
 
-  # GET /spaces/1 or /spaces/1.json
-  def show; end
```

Comment:
suggestion: we can leave these as is, without removing the comment to keep it as close to the default generated scaffold as possible